### PR TITLE
[Forms] Feature #3799 - Duplication

### PR DESCRIPTION
**Pull request by NicolasEYSSERIC**

### CODE REVIEW

File: core-war/src/main/java/org/silverpeas/web/templatedesigner/control/TemplateDesignerSessionController.java [hunk 1]
```diff
@@ -32,6 +32,11 @@
 import org.silverpeas.core.contribution.template.publication.PublicationTemplateException;
 import org.silverpeas.core.contribution.template.publication.PublicationTemplateImpl;
 import org.silverpeas.core.contribution.template.publication.PublicationTemplateManager;
+import org.silverpeas.core.notification.message.MessageNotifier;
```
Comment (silveruser) [inline core-war/src/main/java/org/silverpeas/web/templatedesigner/control/TemplateDesignerSessionController.java]: MINOR Remove this unused import 'org.silverpeas.core.notification.message.MessageNotifier'. [rule](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by silveruser** [comment]
SonarQube analysis reported 1 issue
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by mmoqui** [comment]
Since Java 7, we have now the new class Files to perform file system operations. With the evolution of the JDK, we are going to replace more and more our own utilitarian code by the JDK one.
Could you replace the use of FileUtil by Files please

**Comment by NicolasEYSSERIC** [comment]
OK. I used Files instead of FileUtil.

**Pull request merged**